src/VisualStudio/TestUtilities/IntegrationLog.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+
+namespace Roslyn.VisualStudio.Test.Utilities
+{
+    public sealed class IntegrationLog : IDisposable
+    {
+        private static IntegrationLog _current;
+
+        private string _outputFolder;
+        private string _outputFile;
+        private StreamWriter _outputStream;
+
+        private IntegrationLog()
+        {
+            _outputFolder = Path.Combine(Environment.CurrentDirectory, "VsIntegrationTestLogs");
```

Comment:
xunit has a logging concept; why wouldn't we just use that? Then there's nice support where it appears in test runner output. (This is also using Environment.CurrentDirectory in a helper class, which to me is an automatic "no". :smile:)